site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,413 @@
+
+When closing Citra SDL, the command line counterpart to our normal GUI version that most people will be used to, the window would remain open unintentionally which required a forceful shutdown to close properly. This would result in cleanup tasks, such as video dumping, failing. Which isn’t very nice if you’ve just spent a long time working on a video just to lose it! 
+
+By implementing a graceful shutdown, the window will close normally and a standard cleanup can be utilized. This results in better closing practices for the SDL version of Citra, and ensures that any data is not lost either!
+
+## Separate Windows ([#6177](https://github.com/citra-emu/citra/pull/6177), [#6198](https://github.com/citra-emu/citra/pull/6198)) by [epicboy](https://github.com/ameerj)
+
+Another very long awaited feature, Separate Windows, has arrived in Citra!
+
+Since the conception of Citra in 2014 (yes, that long ago!) users had been asking for a way to split the two screens of the 3DS shown in Citra into two separate windows. This would allow you to move the two screens around to wherever you want on your PC, and to be used on two separate monitors too! Unfortunately, no one was able to do this for a very long time, until [epicboy](https://github.com/ameerj) came along…
+
+The groundwork for this change had already been put in place with a PR [jroweboy](https://github.com/jroweboy) made for [VSync presentation](https://github.com/citra-emu/citra/pull/4940) back in 2019. The vsync presentation PR was created as a way to force enable VSync for everyone from within Citra. In the past, this was disabled by default. 
+For those who force enabled VSync in their driver settings, Citra wasn’t able to disable VSync, which resulted in inconsistent frame limiting. 
+This was picked up by [epicboy](https://github.com/ameerj) and molded to work with multiple windows being displayed by one instance of Citra.
+
+To enable this feature, head to `View -> Screen Layout -> Separate Windows` from within Citra. Launch your game, and from there you can edit the size and location of the windows to wherever you want on your PC!
+
+{{< figure src="screens.png"
+    title="Map on one screen, gameplay on the other!" >}}
+
+### Ask for confirmation when changing games from the game list ([#6186](https://github.com/citra-emu/citra/pull/6186)) by [foghawk](https://github.com/foghawk)
+
+When you run Citra without Single Window Mode enabled, you have the game window and the game list window open at the same time. If you clicked another game in the game list, it’d instantly boot, closing your current game. 
+
+After experiencing this themselves via accidentally clicking onto another game whilst already in a game, [foghawk](https://github.com/foghawk) added a confirmation check. No longer will you lose any important save data from this issue, and you can carry on playing after selecting one of the two options!
```

Comment:
You use "whilst" a lot... usually you just want to use "while"

Reply:
That would be because autumn is British :P
"whilst" isn't used much in America, but it's still used in formal writing in the UK. As a Dutchman, I was taught UK English in education, and "whilst" was often preferred over "while" where possible.

Reply:
(i think this is just british english, but agree we should stick to a coherent style)

Reply:
Yeah, using While instead of Whilst is pretty weird to me... I'll change it anyway though haha